tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/ProjectVersion.java

```diff
@@ -0,0 +1,24 @@
+package org.hawkular.apm.tests;
```

Comment:
You need to add the source code header block to this and other java files. I'm surprised that it built, as the license plugin should have complained.